### DISCUSSION TITLE
Welcome to the World of  Web3 , Blockchain . Discussions!

### DISCUSSION THREAD

**Discussion by Vikash-8090-Yadav** [Vikash-8090-Yadav/Future.WebNet#144] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Omega12Pirme** [comment]
Hey  this is Omega and am Currently doing Btech and currently am in 2nd yr . I love ur repo and would like to contribute in yours under SWOC . I already know Web3 ,Ether js , React . Looking forward to work with u as a contributor.  Thanks ✌️

**Comment by Vikash-8090-Yadav** [comment]
All new issue for the beginner as well for the intermediate created . Go and take  . Totally based on First come First Serve 
Thank u .
